fiftyone/factory/repos/delegated_operation_doc.py

```diff
@@ -102,6 +102,22 @@ def from_pymongo(self, doc: dict):
         self.id = doc["_id"]
         self._doc = doc
 
+        # generated fields:
```

Comment:
I also don't think this needs to be added. The merge conflicts arose because `OperatorRegistry(enabled="all")` was changed to `OperatorRegistry()` on release branches, but that code is no longer needed on `develop` because Trinity is refactoring how this works.